README.md

```diff
@@ -17,6 +17,7 @@ A [Makefile](https://github.com/lightninglabs/faraday/blob/master/Makefile) is p
 ```
 go get -d github.com/lightninglabs/faraday
```

Comment:
better to just use `git clone` instead? cc @guggero

Reply:
Yes, that's what I suggested on Slack as well.